ui_framework/components/modal/modal_overlay.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classnames from 'classnames';
+
+export function KuiModalOverlay({ className,  ...rest }) {
+  const classes = classnames('kuiModalOverlay', className);
+  return (
+    <div
+      className={ classes }
+      { ...rest}
+    />
+  );
+}
+
+KuiModalOverlay.propTypes = {
+  className: React.PropTypes.string,
```

Comment:
More `React.PropTypes`, which we should stop using. Please use the `prop-types` module.